ostd/src/mm/frame/frame_ref.rs

```diff
@@ -40,6 +41,17 @@ impl<M: AnyFrameMeta + ?Sized> Deref for FrameRef<'_, M> {
     }
 }
 
+impl<M: AnyFrameMeta + ?Sized> Clone for FrameRef<'_, M> {
+    fn clone(&self) -> Self {
```

Comment:
```rust
            let pt_cloned = (*pt).clone();
```
So, looks like this is very easy to be misused...

For now, how about adding an associated method called `clone_ref`? I can see the downside: The `#[derive(Clone)]` can't be used if the type contains a `FrameRef`. However, this seems to be the least annoying option, as we don't currently have a case like that.